packages/react-search-ui/src/containers/Facet.js

```diff
@@ -12,6 +12,8 @@ function findFacetValueInFilters(name, filters, filterType) {
   return filter.values;
 }
 
+const accentFold = (str = '') => str.normalize('NFD').replace(/[\u0300-\u036f]/g, "");
```

Comment:
Since this is a fairly terse implementation, would you mind adding some comments about what this does?

Or alternatively, it might be nice to move this to a helper and add some explicit tests around this demonstrating the intended behavior.